components/ds18b20/examples/ds18b20-read/main/ds18b20-read.c

```diff
@@ -43,12 +43,14 @@ static void sensor_detect(void)
         search_result = onewire_device_iter_get_next(iter, &next_onewire_device);
         if (search_result == ESP_OK) { // found a new device, let's check if we can upgrade it to a DS18B20
             ds18b20_config_t ds_cfg = {};
+            onewire_device_address_t address;
             // check if the device is a DS18B20, if so, return the ds18b20 handle
             if (ds18b20_new_device(&next_onewire_device, &ds_cfg, &s_ds18b20s[s_ds18b20_device_num]) == ESP_OK) {
-                ESP_LOGI(TAG, "Found a DS18B20[%d], address: %016llX", s_ds18b20_device_num, next_onewire_device.address);
+                ESP_ERROR_CHECK(ds18b20_get_device_address(s_ds18b20s[s_ds18b20_device_num], &address));
```

Comment:
this check can be removed because we're sure the parameters passed to the function must be valid.

```suggestion
                ds18b20_get_device_address(s_ds18b20s[s_ds18b20_device_num], &address);
```